Squaads/Domain-Driven-Hexagon
# feat: graphql basic implementation

## **Problem**

Describe what functionality or problem this PR is solving.

Ex: Admins couldn't modify the owner of a reservation.

## **Changelog**

What have you changed in this PR and what will happen from now on. You can also explain how the interface behaves perhaps.

Ex: Now the admins have a section on reservation dialog to search another owner to change it or delete it.

## **How to test**

The steps to reproduce this PR and to check its functionality would be the following:

1.  Do this.. (Ex, Create new reservation)

2.  Do that.. (Ex, Change the owner of that reservation)

3.  Check this.. (Ex, Test that as a normal user cant modify the owner of the reservations)

## **Type of change (Optional)**

-   [ ] Refactor (non-breaking change which modifies code keeping the same functionality)
